_config/config.yml

```diff
@@ -10,3 +10,4 @@ Controller:
 
 QuickFeedbackExtension:
   rate_limit: 3
+  redirect_field: false
```

Comment:
I wanted to show the configuration parameter. The class property serves as a default, if this config parameter is removed.